plugins/home/src/homePageComponents/QuickStart/Content.test.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Content } from './Content';
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { renderInTestApp, TestApiProvider } from '@backstage/test-utils';
+import ContentImage from './static/backstageSystemModel.png';
+
+describe('<QuickStartCard />', () => {
+  const Wrapper = ({ children }: { children?: React.ReactNode }) => (
+    <TestApiProvider apis={[]}>{children}</TestApiProvider>
```

Comment:
same here, I don't think you need to use the test api wrapper if you don't have any apis